src/abstractions/configuration.ts

```diff
@@ -6,5 +6,6 @@ export type AppConfiguration = {
   readonly unlayerEditorManifestUrl: string;
   readonly loaderUrl: string;
   readonly dopplerLegacyBaseUrl: string;
+  readonly htmlEditorApiBaseUrl: string;
```

Comment:
I have added the environments configuration in https://github.com/MakingSense/doppler-swarm/pull/591